modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the form view
+
+You can use a similar procedure to customize the following UI items:
+** Quick starts visible to users
+** Cluster roles accessible to a project
+** Actions visible on the +Add page
+
+.Prerequisites
+
+* You must have administrator privileges.
```

Comment:
I also need access to whatever platform I can perform these actions on, correct?

Reply:
By platform you mean?

Reply:
Need more context here @deerskindoll.

Reply:
Fixed.